tools/slow-log-timelines/src/main/java/Timeline.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 TiKV Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import java.time.Duration;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+import slowlog.SlowLog;
+
+public class Timeline implements Callable<Integer> {
+
+  @Option(
+      names = {"--resolution"},
```

Comment:
what does 'resolution' mean?

Reply:
Comment added. The former is the meaning.

Reply:
I prefer renaming it to 'width', what do other tools used for this purpose?